### PR TITLE
Updated realm syntax in p11

### PR DESCRIPTION
**Pull request by zackleman**

### CODE REVIEW

**Comment by extrajordanary** [comment]
@Ben-G

File: P11-Search/content.md [hunk 1]
```diff
@@ -108,7 +111,8 @@ We are setting the default state. However nothing will happen unless we use the
             // update notes and search bar whenever State changes
             switch (state) {
             case .DefaultMode:
-                notes = Note.allObjects().sortedResultsUsingProperty("modificationDate", ascending: false) //1 
+                let realm = Realm()
+                notes = realm.objects(Note).sorted("modificationDate", ascending: false) //1 //1
```
Comment (cocojoe) [inline P11-Search/content.md]: double comment

File: P11-Search/content.md [hunk 2]
```diff
@@ -94,8 +95,10 @@ When the `Dashboard` is presented we want to revert to `.DefaultMode`.
 > Ensure `func viewWillAppear` reads as follows:
 > 
     override func viewWillAppear(animated: Bool) {
-      super.viewWillAppear(animated)
-      state = .DefaultMode
+        super.viewWillAppear(animated)
+        let realm = Realm()
+        notes = realm.objects(Note).sorted("modificationDate", ascending: false)
```
Comment (cocojoe) [inline P11-Search/content.md:100]: filtering then setting state which will repeat this again

**Comment by cocojoe** [comment]
the realm syntax is more inline with docs which is good however this needs tidied up a little and you will need to pull  upstream to fix for auto-merge.  Thx

**Pull request merged**